python/docs/source/development/ps_contributing.rst

```diff
@@ -1,192 +0,0 @@
-==================
-Contributing Guide
-==================
-
-.. contents:: Table of contents:
-   :depth: 1
-   :local:
-
-Types of Contributions
```

Comment:
This section is removed as they are already documented https://spark.apache.org/contributing.html and https://spark.apache.org/docs/latest/api/python/development/contributing.html